library/core/src/char/methods.rs

```diff
@@ -672,8 +673,9 @@ impl char {
     /// 'ß'.encode_utf8(&mut b);
     /// ```
     #[stable(feature = "unicode_encode_char", since = "1.15.0")]
+    #[rustc_const_stable(feature = "const_char_encode_utf8", issue = "130512")]
```

Comment:
```suggestion
    #[rustc_const_unstable(feature = "const_char_encode_utf8", issue = "130512")]
```

This also needs to go on the other function